examples/social/README.md

```diff
@@ -0,0 +1,133 @@
+# A Social Media Example Application
+
+This example illustrates how to use channels for cross-chain messages.
+
+For simplicity, each microchain represents one user—its owner. They can subscribe to other
+users and make text posts that get sent to their subscribers.
+
+## How it Works
+
+The application's state on every microchain contains a set of posts created by this chain
+owner, and a set of posts received from other chains that it has subscribed to. The
+received posts are indexed by timestamp, sender and index.
+
+There are `Subscribe` and `Unsubscribe` operations: If a chain owner includes these in a
+new block, they can subscribe to or unsubscribe from another chain.
+
+There is also a `Post` operation: It creates a new post and sends it to a channel, so that
+it reaches all subscribers.
+
+There are corresponding `RequestSubscribe`, `RequestUnsubscribe` and `Posts` cross-chain
+message variants that are created when these operations are handled. The first two are
+sent directly to the chain we want to subscribe to or unsubscribe from. The latter goes
+to the channel.
+
+## Usage
+
+To try it out, first setup a local network with two wallets, compile the `social` example
+and create an application with it:
+
+```bash
+./scripts/run_local.sh
```

Comment:
I'd suggest mentioning that this script should be executed in a separate terminal, and that it must be kept running for the other commands to succeed.